index.js

```diff
@@ -0,0 +1,129 @@
+import { DateTime } from './modules/luxon.js';
+import Add from './modules/add.js';
+import Remove from './modules/remove.js';
+
+const add = new Add();
+const remove = new Remove();
+// Add current time
+const now = DateTime.now();
+document.querySelector(
+  '.date-now',
+).innerHTML = `${now.day} ${now.month}, ${now.year} ${now.hour}:${now.minute}:${now.second}`;
+
+// eslint-disable-next-line no-undef
```

Comment:
- [ ] Please avoid disabling linter rules, the linters help tremendously to keep a consistent code structure and enforce some best practices over your code! Kindly consider removing this comment and fixing the error. 🤓 🔧